tests/test_slogpy_console.py

```diff
@@ -23,11 +23,58 @@ def test_debug_console_output(capsys, msg=""):
 
     When: Using debug() with a String
 
-    Then: Contents of String shows up in stdout AND nothing in stderr
+    Then: Contents of String shows up in a file AND nothing in stdout and stderr
 
     """
     slog.debug(msg)
     expected_content = ""
-    # not sure this line of code is necessary or not. since this won't be any content in the console.
     out, _ = capsys.readouterr()
-    assert expected_content == ""
+    assert expected_content == out
+
+def test_annoy_console_output(capsys, msg="This is a console test"):
+    """Simple test for console output
+
+    Given:Using slogpy
+    When: Using annoy() with a String
+    Then: Contents of String shows up in stdout AND nothing in stderr
+    """
```

Comment:
annoy()'s promise is to only annoy you once, you are not verifying that it does that.